app/src/main/java/org/imozerov/streetartview/network/FetchService.kt

```diff
@@ -19,6 +20,7 @@ class FetchService : IntentService("FetchService") {
     lateinit var dataSource: DataSource
 
     override fun onHandleIntent(intent: Intent?) {
+        Log.d(TAG, "onHandleIntent($intent)")
         if (intent != null) {
```

Comment:
The injection should be moved to onCreate. It is odd that a field is injected on any intent.